app/src/main/java/com/internship/move/utils/extensions/CustomDialogFragment.kt

```diff
@@ -44,10 +44,10 @@ class CustomDialogFragment : DialogFragment(R.layout.item_custom_dialog) {
 
         private val bundle = Bundle()
```

Comment:
Create a new bundle object in the `newInstance` method. Particularly inside the `apply` block.
Since this variable is shared between multiple instances, it could lead to inconsistent values